### PR TITLE
Add Apache Phoenix images

### PR DESCRIPTION
**Pull request by Praveen2112**

### CODE REVIEW

File: testing/phoenix4/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ARG BASE_URL=https://archive.apache.org/dist
+ARG ZOOKEEPER_BINARY_PATH=$BASE_URL/zookeeper/zookeeper-$ZOOKEEPER_VERSION/zookeeper-$ZOOKEEPER_VERSION.tar.gz
+ARG HBASE_BINARY_PATH=$BASE_URL/hbase/$HBASE_VERSION/hbase-$HBASE_VERSION-bin.tar.gz
+ARG PHOENIX_BINARY_PATH=$BASE_URL/phoenix/phoenix-$PHOENIX_VERSION/phoenix-hbase-$PHOENIX_HBASE_VERSION-bin.tar.gz
+
+RUN yum update -y && \
+    yum install -y gettext && \
+    # Install Zulu JDK
+    echo "Downloading zulu${JAVA8_ZULU_VERSION}-linux.x86_64.rpm..." && \
+    curl -o /tmp/jdk8.rpm --url https://cdn.azul.com/zulu/bin/zulu${JAVA8_ZULU_VERSION}-linux.x86_64.rpm && \
```
Comment (losipiuk) [inline testing/phoenix4/Dockerfile:36]: Do we need 8?

File: testing/phoenix4/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    rm phoenix-hbase-$PHOENIX_HBASE_VERSION-bin.tar.gz && \
+    ln -s $IMAGE_ROOT_INSTALL/phoenix-hbase-$PHOENIX_HBASE_VERSION-bin $PHOENIX_HOME
+
+RUN ln -s $PHOENIX_HOME/phoenix-server-hbase-$PHOENIX_HBASE_VERSION.jar $HBASE_HOME/lib/phoenix.jar
+
+RUN mkdir -p /var/log/zookeeper /var/log/hbase
+
+COPY ./files /
+
+RUN rm $HBASE_HOME/conf/hbase-site.xml && \
```
Comment (losipiuk) [inline testing/phoenix4/Dockerfile:74]: no need for `rm`

File: testing/phoenix5/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    rm phoenix-hbase-$PHOENIX_HBASE_VERSION-bin.tar.gz && \
+    ln -s $IMAGE_ROOT_INSTALL/phoenix-hbase-$PHOENIX_HBASE_VERSION-bin $PHOENIX_HOME
+
+RUN ln -s $PHOENIX_HOME/phoenix-server-hbase-$PHOENIX_HBASE_VERSION.jar $HBASE_HOME/lib/phoenix.jar
+
+RUN mkdir -p /var/log/zookeeper /var/log/hbase
+
+COPY ./files /
+
+RUN rm $HBASE_HOME/conf/hbase-site.xml && \
```
Comment (losipiuk) [inline testing/phoenix5/Dockerfile:58]: no need for `rm`

**Review by losipiuk — APPROVED** [review]

File: testing/phoenix4/Dockerfile (hunk 1 above)
Comment (Praveen2112) [reply testing/phoenix4/Dockerfile:36]: From this documentation - https://hbase.apache.org/book.html#java. HBase 1.4+ doesn't support Java 11. (have tested it locally haven't faced any issues with Java 11). But based on official documentation we could stick to Java 8 for HBase 1.4. WDYT ?

**Comment by Praveen2112** [comment]
@losipiuk AC.

File: testing/phoenix4/Dockerfile (hunk 1 above)
Comment (losipiuk) [reply testing/phoenix4/Dockerfile:36]: Let's keep 8

**Pull request merged**